homeassistant/components/aurora_abb_powerone/__init__.py

```diff
@@ -47,3 +51,61 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
+
+
+class AuroraAbbDataUpdateCoordinator(DataUpdateCoordinator):
+    """Class to manage fetching AuroraAbbPowerone data."""
+
+    def __init__(self, hass: HomeAssistant, comport: str, address: int) -> None:
+        """Initialize the data update coordinator."""
+        self.available_prev: bool = False
+        self.available: bool = False
```

Comment:
```suggestion
        self.available_prev = False
        self.available = False
```

Reply:
mypy can figure it out since you are setting it to a bool

Reply:
Ok, done 6eae5b3